http/headers/alt-svc.json

```diff
@@ -0,0 +1,105 @@
+{
+  "http": {
+    "headers": {
+      "Alt-Svc": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Alt-Svc",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": [
+              {
+                "version_added": "38"
+              },
+              {
+                "version_added": "37",
```

Comment:
I'm thinking that maybe we should add some `version_removed` statements in these blocks.